### DISCUSSION TITLE
What does battery working mode "Adaptive" do?

### DISCUSSION THREAD

**Discussion by PulsarFX** [wlcrs/huawei_solar#560] [General]
In the dropdown for the working mode of the battery, there is "Adaptive". I can't find this mode in any huawei docs. 
This sensor is called `select.battery_working_mode`

![grafik](https://github.com/wlcrs/huawei_solar/assets/4960210/8dd6822e-d699-45a0-98bb-317ef495785e)

Can anyone explain what this mode does and how it is supposed to be used?

**Comment by LotharWoman** [comment]
I´m also very interesstet.
